packages/waas/package.json

```diff
@@ -18,6 +18,7 @@
     "@aws-sdk/client-cognito-identity-provider": "^3.445.0",
     "@aws-sdk/client-kms": "^3.429.0",
```

Comment:
can we remove `@aws-sdk/client-kms` or any of the aws packages now that we've removed the transport key..? just noting we should purge any unused deps from here.